File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/gallery/GalleryView.java
package ua.com.amicablesoft.android.wr.ui.gallery;

import java.util.ArrayList;

import ua.com.amicablesoft.android.wr.models.Powerlifter;

/**
 * Created by olha on 2/16/17.
 */

interface GalleryView {

    void setListPowerlifters(ArrayList<Powerlifter> list);
    void setPowerlifter(int position);

    void setListItems(ArrayList<String> items);

    void showLoading();
    void dismissLoading();
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/gallery/GalleryView.kt
/*
 * Created by Olha Tymoshenko on 5/25/18 1:37 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.ui.gallery

import ua.com.amicablesoft.android.wr.models.Powerlifter
import java.util.*

internal interface GalleryView {

    fun setListPowerlifters(list: ArrayList<Powerlifter>)
    fun setPowerlifter(position: Int)

    fun setListItems(items: ArrayList<String>)

    fun showLoading()
    fun dismissLoading()
}
